src/Chapter02/Listing02.22.ACheckedBlockExample.cs

```diff
@@ -8,15 +8,15 @@ public static void Main()
             #region HIGHLIGHT
             checked
             {
-            #endregion
+                #endregion HIGHLIGHT
```

Comment:
```suggestion
            #endregion HIGHLIGHT
```
nit: remove whitespace